azure_arc_k8s_jumpstart/aksengine/arc_k8s_aksengine.azcli

```diff
@@ -0,0 +1,432 @@
+########################################################
```

Comment:
Please check other shell scripts to align format: 

https://github.com/microsoft/azure_arc/blob/main/azure_arc_servers_jumpstart/scripts/az_connect_linux.sh

Reply:
awesome!